app/serializers/user_topic_bookmark_serializer.rb

```diff
@@ -59,6 +59,10 @@ def bookmarkable_url
     topic.url
   end
 
+  def url_for_ui
```

Comment:
In seems like we can just rely on `bookmarkable_url`?

Reply:
`bookmarkable_url` is different because it links to the first post in the topic but in the UI we want to link to the first unread post.

Reply:
I think we can just pass in an option to the serializer instead in cases where we want the bookmarkable_url to point to the last unread. That way, we don't have to ship an extra attribute and can just reply only a single property on the client side.

Reply:
this is done!